script/make_report_time.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 shift  # get rid of the '-c' supplied by make.
 
-\time -o bench.txt -a -f "[%E] : $*" -- sh -c "$*"
+if command time --version > /dev/null 2>&1
```

Comment:
Unfortunately this does not work in zsh.

```shell
command time --version

time: illegal option -- -
usage: time [-lp] command.
```

I'm for dropping this feature altogether and let people opt in by setting on SHELL instead of this being in the Makefile:
https://github.com/3scale/apisonator/blob/2aef6655f3a41dcdd99c3fc015fb5778d728a405/Makefile#L3

Reply:
@mikz this should be interpreted by /bin/sh not zsh, working fine for me, with zsh.

Reply:
I see! 👍